adhocracy4/datetimefield/widgets.py

```diff
@@ -0,0 +1,78 @@
+import datetime
+
+from django.contrib.staticfiles.storage import staticfiles_storage
+from django.forms import widgets as form_widgets
+from django.template.loader import render_to_string
+from django.utils.timezone import localtime
+
+
+class DateTimeInput(form_widgets.SplitDateTimeWidget):
+    def __init__(self, time_label='', time_default=None, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.time_label = time_label
+        self.time_default = time_default or datetime.time(hour=0, minute=0)
+
+    class Media:
+        js = (
+            staticfiles_storage.url('datepicker.js'),
```

Comment:
Are those file parts of adhocracy4? Should they be?

Reply:
This one might be a deal breaker. We searched long and hard for a good datepicker, and found none. In the end vellip ended up packaging one that has been unmaintained since 2014 at https://github.com/vellip/datePicker so it could be used with npm/webpack.

Our setup is to install it from github via npm, and then bundle it together with an initialisation script into datepicker.js via webpack.

Maybe it's best if we just bypass all that here and just add documentation: This widget will create a regular text input with the class `datepicker`. It will also load `datepicker.js` and `datepicker.css`. The individual projects are in charge of creating these files and implementing the actual datepicker.

Reply:
If this is the final solution, we should use our own fork of that datepicker.